examplesRSECTION/high_level_functions_demos/basic_objects_demo.js

```diff
@@ -0,0 +1,101 @@
+if (!RSECTION) {
+    throw new Error("This script is only for RSECTION.");
+}
+
+include("../includes/Tools/high_level_functions_support.js");
+
+/*********************************************************************************************
+****************************************** Main **********************************************
+*********************************************************************************************/
+
+var t1 = new Date().getTime();
+
+run("../includes/Tools/clearAll.js");
+
+var material = createMaterial("S235");
```

Comment:
Could be handy to have also some HFL for Materials and Sections

Reply:
Sections are covered, but Materials not (I think there are some missing (un-complete) API support for concrete material values, timber materials values and so on).